services/wallet/token/token-lists/builder.go

```diff
@@ -125,7 +125,11 @@ func filterTokens(tokens []*tokenTypes.Token) []*tokenTypes.Token {
 				break
 			}
 		}
-		if !found {
+		tokenOnBscChain := token.ChainID == common.BSCMainnet || token.ChainID == common.BSCTestnet
+		// remove native token on respective chains as they are added via a different list
+		if !found ||
```

Comment:
we have function `func (t *Token) IsNative() bool` https://github.com/status-im/status-go/blob/develop/services/wallet/token/types/token.go#L42